documentation/docs/reference/software/interfaces/mqtt.md

```diff
@@ -59,171 +109,390 @@ Control of the focus stage. The message is a JSON object, speed is optional:
+|---------------------------------------------------|-----------------------------------------------------------------------------------------------------------|
+| `"Camera settings updated"`         | The camera settings have been successfully updated.                                                       |
+| `"Camera settings error"`           | The settings command is missing required parameters.                                                      |
+| `"Error: Resolution not valid"`     | The provided resolution parameter is invalid.                                                             |
+| `"Error: Iso number not valid"`     | The provided ISO parameter is invalid.                                                                    |
+| `"Error: Shutter speed not valid"`  | The provided shutter speed parameter is invalid.                                                          |
+| `"Error: White balance gain not valid"` | The provided white balance gain parameters are invalid.                                                   |
+| `"Error: White balance mode %s not valid"` | The provided white balance mode parameter is invalid.                                                      |
+| `"Error: Image gain not valid"`     | The provided image gain parameters are invalid.                                                           |
+| `"Busy"`                            | The camera is currently busy and cannot update settings.                                                  |
 
-State of the imager. It's a JSON object with:
+- **JSON message to stop the imager**:
 
 ```json
 {
-  "status": "Started",
-  "time_left": 25
+  "action": "stop"
 }
 ```
 
-Status is one of `Started`, `Ready`, `Completed` or `12_11_15_0.1.jpg has been imaged`.
+**Authorized values for `stop` action:**
 
-- Publish only
+| Field    | Type   | Accepted Values |
+|----------|--------|-----------------|
+| `action` | string | "stop"          |
 
-#### `status/segmenter`
 
-Status of the segmentation. It's a JSON object with:
+**Status/Error Messages for `stop` action:**
 
-```json
-{
-  "status": "Started"
-}
-```
+| Status/Error message                                 | Description                                                                                                  |
+|---------------------------------------------------|-----------------------------------------------------------------------------------------------------------|
+| `"Interrupted"`                     | The image capture process was stopped successfully.                                                       |
+| `"Busy"`                            | The camera is currently busy and cannot stop the operation.                                               |
 
-`status` is one of `Started`, `Done`, `Interrupted`, `Busy`, `Ready` or `Dead`.
 
-- Publish only
 
-#### `status/segmenter/object_id`
```

Comment:
You have deleted any description of the `status/segmenter/object_id`, `status/segmenter/metric`, etc., status messages. This page needs to describe those messages.